commons/src/test/java/gov/cms/qpp/conversion/util/EnvironmentHelperTest.java

```diff
@@ -1,15 +1,17 @@
 package gov.cms.qpp.conversion.util;
 
+import static com.google.common.truth.Truth.assertThat;
 import static com.google.common.truth.Truth.assertWithMessage;
 
 import java.util.Properties;
 import java.util.UUID;
 
+import gov.cms.qpp.test.logging.LoggerContract;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
-class EnvironmentHelperTest {
+class EnvironmentHelperTest implements LoggerContract {
```

Comment:
Sorry, I should have caught this earlier, but we should also have a test for when the env var is present but an empty string.  I believe all the existing tests just use the precondition that the env var doesn't exist.